packer/gpu/buildkite-gpu-ami.pkr.hcl

```diff
@@ -0,0 +1,42 @@
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+
+source "amazon-ebs" "gpu_box" {
+  ami_name        = "buildkite-stack-linux-gpu-${local.timestamp}"
+  ami_description = "Buildkite Elastic Stack (Amazon Linux 2 LTS w/ nvidia-docker)"
```

Comment:
should we call it with `vLLM` in the name? like `vLLM Buildkite Stack`

Reply:
done